server/api.go

```diff
@@ -27,6 +27,9 @@ func newHandler(plugin *Plugin) *handler {
 	root := mux.NewRouter()
 	root.Use(handler.authorized)
 
+	tsdb := root.PathPrefix("/tsdb").Subrouter()
+	tsdb.HandleFunc("", handler.getStatsHandler).Methods(http.MethodGet)
```

Comment:
0/5 maybe we can add a /stats prefix here in case we might want to add something more tsdb related later?

Reply:
Sure, can do that.